recipes/funchook/all/conanfile.py

```diff
@@ -0,0 +1,114 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+required_conan_version = ">=1.33.0"
+
+
+class FunchookConan(ConanFile):
+    name = "funchook"
+    homepage = "https://github.com/kubo/funchook"
+    description = " Hook function calls by inserting jump instructions at runtime"
+    topics = ("function", "hooking")
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "GPL-2.0-or-later-linking-exception"
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "disassembler": ["diStorm3", "zydis", "capstone"],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "disassembler": "diStorm3",
```

Comment:
Distorm is a git submodule, so it should be used as a separated package first.

Reply:
Aha, good point!

QQ regarding proper procedure.  Is it acceptable to add 2 packages in the same PR?  

If so, thinking I'd just rewrite the history prior to final merge where we'd have 2 commits - one for adding diStorm3 and one for adding funchook?  Or should this be 2 PRs?

Reply:
It must be 2 separated PRs. The CI bot is one programmed to validate only one project per PR, if you add more than one, it will be automatically blocked.